modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -206,6 +212,60 @@ export default function AuditBarChart({rtl, vocabularies}) {
 
 	const [tooltip, setTooltip] = useState(null);
 
+	const onBarClick = (assetCategoryIds) => {
+		if (assetCategoryIds.length) {
+			let uri = window.location.href;
+
+			if (!uri.includes('resetBarsCategoryFiltersURL')) {
+				uri = Liferay.Util.addParams(
+					'resetBarsCategoryFiltersURL=' + encodeURIComponent(uri),
+					uri
+				);
+			}
+
+			const params = new URLSearchParams(window.location.search);
+
+			const categories = params.getAll(namespace + 'assetCategoryId');
+
+			if (categories.length) {
+				categories.map((category) => {
+					uri = uri.replace(
+						namespace + 'assetCategoryId=' + category,
+						''
+					);
+				});
```

Comment:
You are using a `.map` and not returning anything in the map or setting this to a variable, which are two indicators that its better to use `.forEach` in this case.

Reply:
Sure!